### PR TITLE
#439 Updated branch definition

### LINKED ISSUE #439: [Content Bug] Response for week 1 step 1
**Identify Steps:**
* Step _1

**Week:**

week 1

### Branch Definition

The definition of a [branch](https://github.com/bitprj/Intro-To-Serverless/blob/main/.bit/responses/1.1-Week%201%20Step%201.md#%EF%B8%8F-branch) is not correct imho when it comes to the comparison with a [fork](https://github.com/bitprj/Intro-To-Serverless/blob/main/.bit/responses/1.1-Week%201%20Step%201.md#%EF%B8%8F-fork)

A fork is a copy of repository (including its branches) that can (but need not) be used to contribute to a project.

A branch is created within one repository with the goal to create another line of development that is isolated from the stable (main) branch. This enables the changes of code in an isolated environment without destabilizing the main branch. 

### Typo in Pull request Section

In the [Pull request section](https://github.com/bitprj/Intro-To-Serverless/blob/main/.bit/responses/1.1-Week%201%20Step%201.md#%EF%B8%8F-pull-request) there seems to be a "typo":

>>*You are basically asking for permission to "merge" your changes."

I guess the star as well as the last quotes are not needed. 

### Enhancement

Maybe a short description of the typical flow to sync your fork with the upstream repository via GitHub might be of value for the students:

Fetch + Merge upstream -> Pull changes to your local repo ->  commit changes to branch -> psuh branch 

## Proposed Fix

- Remove the comparison with a fork from the branch definition.
- Correct typo
- (Optional) Add GH Flow with forks

### PR DESCRIPTION
**Pull request by blueboxes**
## Changes made
* Text update to definitions

Closes #439

### CODE REVIEW

**Review by emsesc — CHANGES_REQUESTED** [review]

File: .bit/responses/1.1-Week 1 Step 1.md [hunk 1]
```diff
@@ -48,10 +48,10 @@ For our camp, each week is placed inside an issue. Only when you complete the we
 If you want to contribute to someone else's code, you would "fork" it. This creates a copy of the code under your account that you can make changes to. Create a fork when you **want to make changes to someone else's code and contribute to it.**
 
 #### ‼️ Branch
-Creating a **branch** on a repository is like forking a repository. You would do this when you **want to make changes to your code without harming a working version.**
+Creating a new **branch** on a repository is like create another line of development that is isolated from other code. You would do this when you **want to make changes to your code without effecting a working version.**
```
Comment (ganning127) [inline .bit/responses/1.1-Week 1 Step 1.md:51]: is like "creating another line..."

**Review by ganning127 — CHANGES_REQUESTED** [review]

File: .bit/responses/1.1-Week 1 Step 1.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lechnerc77) [reply .bit/responses/1.1-Week 1 Step 1.md:51]: I am struggeling a bit with the word "isolated" as a branch is not isolated, but still attached to the main branch. Perhaps ".. is like creating a new version or another line of development of your repository"

File: .bit/responses/1.1-Week 1 Step 1.md [hunk 2]
```diff
@@ -48,10 +48,10 @@ For our camp, each week is placed inside an issue. Only when you complete the we
...
 
 #### ‼️ Pull Request
-Once you make changes on **a forked repository or another branch,** you might need to bring the changes into the "main" repository. This allows YOUR changes to be visible in the main project! *You are basically asking for permission to "merge" your changes."
+Once you make changes on **a forked repository or another branch,** you might need to bring the changes into the `main` repository. This allows YOUR changes to be visible in the main project! *You are basically asking for permission to "merge" your changes.*
```
Comment (lechnerc77) [inline .bit/responses/1.1-Week 1 Step 1.md:54]: Proposal for wording: 
"Once you make changes on **a fork or a branch**, you might want to bring the changes into the *main* branch of the branched/forked repository. ..."

**Review by lechnerc77 — CHANGES_REQUESTED** [review]

**Comment by blueboxes** [comment]
@lechnerc77  @ganning127  thanks for your feedback, I have updated based on the comments (sorry for the delay)

**Review by lechnerc77 — APPROVED** [review]
Looks good!

**Pull request closed without merging**